### PR TITLE
NetBSD requires <stdarg.h> for variable argument lists in output.inl

### PR DESCRIPTION
**Pull request by krytarowski**

### CODE REVIEW

**Comment by jasonwilliams200OK** [comment]
:+1:

**Comment by jkotas** [comment]
I think that these #include are commented out for a reason to avoid conflicts with PAL own definitions (PAL is essentially a partial Windows SDK emulator, and so it has conflicting definitions in cases when there is a difference between Unix and Windows).

Could you please try to move this #include to the .c files that are failing to compile to check whether it will fix the problem in #2937?

**Comment by krytarowski** [comment]
Done!

File: src/pal/src/safecrt/safecrt_output_s.c [hunk 1]
```diff
@@ -41,4 +41,5 @@ typedef char        _TCHAR;
 typedef char        TCHAR;
 #define _T(x)       x
 
+#include <stdarg.h>
```
Comment (jkotas) [inline src/pal/src/safecrt/safecrt_output_s.c]: Could you please move it up right after `<stdlib.h>`?

Comment (krytarowski) [inline src/pal/src/safecrt/safecrt_output_s.c]: OK!

My intention was to show you that code using `wchar_t`/`char` is `typedef`ed later. I will do it.

**Comment by krytarowski** [comment]
New version pushed.

**Comment by jkotas** [comment]
LGTM

**Pull request merged**